itk_dev_shared_components/eflyt/eflyt_case.py

```diff
@@ -0,0 +1,109 @@
+"""Module for handling cases in eFlyt"""
+from dataclasses import dataclass
+from datetime import date, datetime
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+
+
+@dataclass
+class Case:
+    """A dataclass representing an Eflyt case."""
+    case_number: str
+    deadline: date | None
+    case_types: list[str]
+
+
+@dataclass
+class Inhabitant:
+    """A dataclass representing an inhabitant."""
+    cpr: str
+    name: str
+    move_in_date: date
+    relations: list[str]
+
+
+@dataclass
+class Applicant:
+    """A dataclass representing an applicant."""
+    cpr: str
+    name: str
+
+
+def get_beboere(browser: webdriver.Chrome) -> list[Inhabitant]:
+    """Count the number of beboere living on the address.
+
+    Args:
+        browser: The webdriver browser object.
+
+    Returns:
+        The number of beboere on the address."""
+    change_tab(browser, 1)
+    beboer_table = browser.find_element(By.ID, "ctl00_ContentPlaceHolder2_ptFanePerson_becPersonTab_GridViewBeboere")
+    rows = beboer_table.find_elements(By.TAG_NAME, "tr")
+
+    # Remove header
+    rows.pop(0)
+
+    inhabitants = []
+    for inhabitant in rows:
+        moving_in = datetime.strptime(inhabitant.find_element(By.XPATH, "td[1]/span | td[1]/a").text, "%d-%m-%Y").date()
+        cpr = inhabitant.find_element(By.XPATH, "td[2]").text.replace("-", "")
+        name = inhabitant.find_element(By.XPATH, "td[3]").text
+        try:
+            relations = inhabitant.find_element(By.XPATH, "td[4]/span").text.split("<br>")
+        except NoSuchElementException:
```

Comment:
This feels a little iffy. There must be a better way to check for an element's existence.

Reply:
I agree, but I haven't found a better method. Any suggestions?

Reply:
```suggestion
        elements = browser.find_elements(By.XPATH, "td[4]/span")
        if elements:
            relations = elements[0].text.split("<br>")
        else:
            relations = []
```